6/run-demo-clients.sh

```diff
@@ -0,0 +1,5 @@
+./run-client-docker.sh 5001 &
+./run-client-docker.sh 5002 &
+./run-client-docker.sh 5003 &
+./run-client-docker.sh 5004 &
+./run-client-docker.sh 5005
```

Comment:
Ah, in this case it makes more sense to have just one API and a vegeta script hammering it.